datafusion/physical-expr/src/aggregate/array_agg.rs

```diff
@@ -96,6 +109,139 @@ impl AggregateExpr for ArrayAgg {
     fn name(&self) -> &str {
         &self.name
     }
+
+    fn groups_accumulator_supported(&self) -> bool {
```

Comment:
I dont see this function used anywhere... 🤔

Reply:
It's called here datafusion/physical-plan/src/aggregates/row_hash.rs:393